mico-admin/src/app/api/api.service.ts

```diff
@@ -986,4 +1012,116 @@ export class ApiService {
             filter(data => data !== undefined)
         );
     }
+
+
+    // =======
+    // POLLING
+    // =======
+
+
+    /**
+     * Polls the deployment job of an given application and provides feedback if the deployment failed/ was successful.
+     * @param applicationShortName applicationShortName of the application to be polled
+     * @param applicationVersion applicationVersion of the application to be polled
+     */
+    pollDeploymentJobStatus(applicationShortName: string, applicationVersion: string) {
+        const resource = 'poll/jobs/' + applicationShortName + '/' + applicationVersion + '/status';
+        const stream = this.getStreamSource<any>(resource);
+
+        // poll status, end polling after 3 minutes
+        const subPolling = interval(500).pipe(takeUntil(timer(3 * 60 * 1000)))
+            .subscribe(() => {
+                this.getJobStatus(applicationShortName, applicationVersion);
+            });
+
+        // handle incomming status updates
+        const subJobStatus = this.getJobStatus(applicationShortName, applicationVersion).subscribe(newStatus => {
```

Comment:
maybe refactor this method to return the value of `this.getJobStatus` and calling a `startPollingJobStatus` method internally. This would match better with the polling behaviour for application deployment status.